contrib/i3status-rust/template.py

```diff
@@ -0,0 +1,38 @@
+pkgname = "i3status-rust"
+pkgver = "0.32.3"
+pkgrel = 0
+build_style = "cargo"
+# Enable the notmuch feature
+make_build_args = ["--features=notmuch"]
+make_install_args = ["--features=notmuch"]
+make_check_args = ["--features=notmuch"]
+hostmakedepends = ["cargo", "pkgconf"]
+makedepends = [
+    "libpulse-devel",
+    "libsensors-devel",
+    "notmuch-devel",
+    "openssl-devel",
+    "rust-std",
+]
+pkgdesc = "Feature-rich replacement for i3status"
```

Comment:
pkgdesc should say what it does, not what it's a replacement of

Reply:
changed pkgdesc to say what it does